homework_11_11_22/Three.java

```diff
@@ -0,0 +1,29 @@
+import java.util.Scanner;
+
+public class Three {
```

Comment:
in this task you need to get a number from input stream and print sum of all values from 1 to this number, for example if your number is 5 sum you need to sum 1+2+3+4+5